app/src/main/java/org/mozilla/fenix/collections/CollectionCreationUIView.kt

```diff
@@ -208,9 +208,8 @@ class CollectionCreationUIView(
                 val drawable = view.context.getDrawable(R.drawable.ic_new)
                 drawable?.setTint(ContextCompat.getColor(view.context, R.color.photonWhite))
                 view.bottom_bar_icon_button.setImageDrawable(drawable)
-                view.bottom_bar_icon_button.setOnClickListener {
-                    actionEmitter.onNext(CollectionCreationAction.AddNewCollection)
-                }
+                view.bottom_bar_icon_button.contentDescription = null
```

Comment:
This code change seems redundant with the XML change and actually performs a redundant UI change that costs a couple CPU cycles. Is there any reason we need both?

Reply:
I'm not sure if it's needed; I saw that pattern too, same changes in the XML and programmatically for the same attributes (ImageButton's drawable, tint, text, etc) so I followed it.

Do you want me to remove one of them (XML or programmatically)?

Reply:
@colintheshots I'd be glad to know your thoughts about this ⏫

Reply:
Yes, please remove the programmatic change. The XML should apply when the View gets created.